projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-interface/wpcom-admin-interface.php

```diff
@@ -417,9 +422,31 @@ function wpcom_is_duplicate_views_experiment_enabled() {
 	if ( ( new Host() )->is_wpcom_simple() ) {
 		\ExPlat\assign_current_user( $aa_test_name );
 		$is_enabled = 'treatment' === \ExPlat\assign_current_user( $experiment_name );
+
+		if ( is_automattician() ) {
+			$is_enabled = true;
+			update_user_option( get_current_user_id(), RDV_EXPERIMENT_FORCE_ASSIGN_OPTION, 'treatment', true );
+		}
+
 		return $is_enabled;
 	}
 
+	$is_proxy_atomic    = defined( 'AT_PROXIED_REQUEST' ) && AT_PROXIED_REQUEST;
+	$is_support_session = WPCOMSH_Support_Session_Detect::is_probably_support_session();
```

Comment:
To fix the Phan error you can do what I did in my previous exploration: https://github.com/Automattic/jetpack/pull/41157/files#diff-83c797b36007d4bbc95f7a8da5f7a750e9fe16930f8a3b7752da09d52899c4b2

Reply:
Done! Thanks!